src/main/java/io/cdap/e2e/utils/BigQueryClient.java

```diff
@@ -27,9 +27,9 @@
 import java.util.Optional;
 
 /**
- * Represents GcpClient
+ * Represents BigQuery Client
  */
-public class GcpClient {
+public class BigQueryClient {
```

Comment:
This change will effect SAP ODP PR as well, https://github.com/data-integrations/sap/pull/10.